docs/StardustDocs/topics/apiLevels.md

```diff
@@ -6,7 +6,7 @@ By nature data frames are dynamic objects, column labels depend on the input sou
 or deleted while wrangling. Kotlin, in contrast, is a statically typed language and all types are defined and verified
 ahead of execution. That's why creating a flexible, handy, and, at the same time, safe API to a data frame is tricky.
 
-In `Kotlin DataFrame` we provide four different ways to access columns, and, while they are essentially different, they
+In Kotlin DataFrame library we provide four different ways to access columns, and, while they are essentially different, they
```

Comment:
Good for clarity, but actually an non-linked up `Kotlin DataFrame` should indicate that we're talking about the library and not the type.

Reply:
Naming and type are so close, and I decided to differ it better with the "library" word